README.md

```diff
@@ -61,6 +61,24 @@ Install smtpapi-nodejs and its dependencies:
 npm install
 ```
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
If you are doing this step, then all you need to do is open and edit the file by inserting your API KEY.

The next section `environment variable` is redundant.